apps/service-portal/src/hooks/useRoutes/useRoutes.ts

```diff
@@ -3,12 +3,14 @@ import { useEffect } from 'react'
 import { ServicePortalModule } from '@island.is/service-portal/core'
 import { Action, ActionType } from '../../store/actions'
 import { useModuleProps } from '../useModuleProps/useModuleProps'
+import { useModules } from '../useModules/useModules'
 import { User } from 'oidc-client'
 import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
 import flatten from 'lodash/flatten'
 
 export const useRoutes = () => {
-  const [{ modules }, dispatch] = useStore()
+  const [_, dispatch] = useStore()
+  const { modules } = useModules()
```

Comment:
In the file above you fetch the modules from the store, using the `useStore` hook.
Which one do we want to use?

Reply:
Ahh I see, it's in the `useModules` hook itself.

On closer inspection, I see that you're following the way it's done for the other hooks. So you can ignore my comment :)

Reply:
Good catch I switched this out to use the store instead of the useModules hook